package.json

```diff
@@ -30,10 +31,12 @@
     "typescript": "^4.1.5"
   },
   "dependencies": {
+    "@octokit/rest": "^20.0.2",
     "bufferutil": "^4.0.3",
     "discord-api-types": "^0.31.0",
     "discord.js": "^12.4.0",
     "discord.js-commando": "^0.12.2",
+    "prettier": "^3.2.5",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
I use prettier for formatting `input-locations.json` after changing it, so no